metrics/src/vespa/metrics/metricmanager.cpp

```diff
@@ -27,6 +27,11 @@ typedef MetricsmanagerConfig Config;
 MetricManager::ConsumerSpec::ConsumerSpec() = default;
 MetricManager::ConsumerSpec::~ConsumerSpec() = default;
 
+time_t
+MetricManager::Timer::getTime() const {
+    return std::chrono::duration_cast<std::chrono::seconds>(std::chrono::steady_clock::now().time_since_epoch()).count();
```

Comment:
Metric snapshots are tagged with the returned timestamp, so they will now receive a value relative to an unknown epoch rather than the UNIX epoch. From a cursory look at the code I don't _think_ this should matter, since it's predominantly used for computing time deltas (where a steady clock is vastly better anyway). But for safety's sake let's replace this one with `system_clock` and then with `steady_clock` after a closer code audit (and maybe also cleanup).

Reply:
Fixed